orchestra/orchestra_api.py

```diff
@@ -89,24 +89,40 @@ def get_project_information(project_ids):
 
 def create_todos(todos):
     response = _make_api_request('post', 'todo-api',
+                                 headers={'Content-type': 'application/json'},
                                  data=json.dumps(todos))
     return json.loads(response.text)
 
 
-def get_todos(project_id, step_slug=None):
+def _convert_filters_to_query_params(filters_dict):
```

Comment:
May be loop over `additional_data` keys to make sure we dont convert filters that are not part of the additional data?
This might solve the problem of injection that @marcua talked about.

Reply:
Currently it ignores filters like {'additional_data__some_nested_field': 'value'}` 😛

Reply:
May be I am missing something. How is it ignoring filters like `{'additional_data__some_nested_field': 'value'}`?

Reply:
Consider using this:
```
import urllib.parse
dict = {
    "title": "Hello world",
    "anonymous": False,
    "needautocategory": True
}
urllib.parse.urlencode(dict)
```